e2e-test/src/test/java/gov/cms/ab2d/e2etest/TestRunner.java

```diff
@@ -0,0 +1,571 @@
+                .GET()
+                .build();
+    }
+
+    private CompletableFuture<HttpResponse<String>> exportByContractRequestAsync(String contractNumber) {
+        HttpRequest exportRequest = buildExportByContractRequest(contractNumber);
+
+        return httpClient.sendAsync(exportRequest, HttpResponse.BodyHandlers.ofString());
+    }
+
+    private HttpResponse<String> statusRequest(String url) throws IOException, InterruptedException {
+        HttpRequest statusRequest = buildStatusRequest(url);
+
+        return httpClient.send(statusRequest, HttpResponse.BodyHandlers.ofString());
+    }
+
+    private HttpRequest buildStatusRequest(String url) {
+        return HttpRequest.newBuilder()
+                .uri(URI.create(url))
+                .timeout(Duration.ofSeconds(DEFAULT_TIMEOUT))
+                .header("Content-Type", "application/json")
+                .header("Authorization", "Bearer " + jwtStr)
+                .GET()
+                .build();
+    }
+
+    private CompletableFuture<HttpResponse<String>> statusRequestAsync(String url) {
+        HttpRequest statusRequest = buildStatusRequest(url);
+
+        return httpClient.sendAsync(statusRequest, HttpResponse.BodyHandlers.ofString());
+    }
+
+    private HttpResponse<String> cancelJobRequest(String jobId) throws IOException, InterruptedException {
+        HttpRequest cancelRequest = HttpRequest.newBuilder()
+                .uri(URI.create(AB2D_API_URL + "Job/" + jobId + "/$status"))
+                .timeout(Duration.ofSeconds(DEFAULT_TIMEOUT))
+                .header("Content-Type", "application/json")
+                .header("Authorization", "Bearer " + jwtStr)
+                .DELETE()
+                .build();
+
+        return httpClient.send(cancelRequest, HttpResponse.BodyHandlers.ofString());
+    }
+
+    private HttpResponse<String> fileDownloadRequest(String jobId, String fileName) throws IOException, InterruptedException {
+        HttpRequest fileDownloadRequest = HttpRequest.newBuilder()
+                .uri(URI.create(AB2D_API_URL + "Job/" + jobId + "/file/" + fileName))
+                .timeout(Duration.ofSeconds(DEFAULT_TIMEOUT))
+                .header("Content-Type", "application/json")
+                .header("Authorization", "Bearer " + jwtStr)
+                .GET()
+                .build();
+
+        return httpClient.send(fileDownloadRequest, HttpResponse.BodyHandlers.ofString());
+    }
+
+    private String getJobUuid() throws SQLException {
+        String uuid = null;
+        try (Connection connection = dataSource.getConnection(); Statement statement = connection.createStatement()) {
+            ResultSet resultSet = statement.executeQuery("SELECT job_uuid FROM job WHERE job_status != 'CANCELLED'");
+            while (resultSet.next()) {
+                uuid = resultSet.getString("job_uuid");
+            }
+        }
+
+        return uuid;
+    }
+
+    private Set<String> getAllJobUuids() throws SQLException {
```

Comment:
I think any code that touches the DB directly might need to be commented out and refactored to be using HTTP interactions.